### PR TITLE
Flow fix

### PR DESCRIPTION
**Pull request by AlexeyBeley**
Minor flow fixes and log

### CODE REVIEW

File: lambda_function.py [hunk 1]
```diff
@@ -202,10 +210,12 @@ def create_stack_set_flow(self) -> None:
 
         try:
             self.create_stack_set()
+            logger.error(f"StackSet '{self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME}' created")
```
Comment (erez-dome9) [inline lambda_function.py:213]: should be info not error

File: lambda_function.py [hunk 2]
```diff
@@ -202,10 +210,12 @@ def create_stack_set_flow(self) -> None:
...
         except Exception as e:
-            if "NameAlreadyExistsException" in repr(e):
+            repr_e_lower = repr(e).lower()
+            if "already" in repr_e_lower and "exists" in repr_e_lower:
                 logger.error(
-                    f"Received NameAlreadyExistsException for stack_set '{self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME}'. Error: {repr(e)}")
+                    f"StackSet '{self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME}' already exists. Skipping this step.")
```
Comment (erez-dome9) [inline lambda_function.py:218]: maybe we should make in warning rather than error 
it is confusing in the CW logs

File: lambda_function.py (hunk 1 above)
Comment (AlexeyBeley) [reply lambda_function.py]: done

File: lambda_function.py (hunk 2 above)
Comment (AlexeyBeley) [reply lambda_function.py:218]: done

**Pull request merged**